CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 CORTX is about building the world's best scalable mass-capacity object storage system. If you’re interested in what we’re building and intrigued by hard challenges, here's everything you need to know about contributing to the project and how to get started. 
 
-This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/1.0.4/CORTX_on_Open_Virtual_Appliance.rst).
+This guide is intended to provide quick start instructions for developers who want to build, test, and contribute to the CORTX software.  If you are merely looking to _test_ CORTX, please use [these instructions](doc/ova/latest/CORTX_on_Open_Virtual_Appliance.rst).
 

```

Comment:
@hessio, don't create a separate file for latest as least OVA release is in https://github.com/Seagate/cortx/blob/e9fa271e318babc3a9c20467adf38259902b62f3/doc/ova/2.0.0/PI-3/CORTX_on_Open_Virtual_Appliance_PI-3.rst

Reply:
@mukul-seagate11, can you check [this](https://github.com/Seagate/cortx/discussions/1185) and maybe put your thoughts to this discussion then?